postcss.config.js

```diff
@@ -1,17 +1,6 @@
-const purgecss = require("@fullhuman/postcss-purgecss");
-
 module.exports = {
   plugins: [
-    purgecss({
-      content: ["./**/*.js"],
-      safelist: {
-        deep: [
-          /su-bg-[A-Za-z0-9]*(-)?([A-Za-z0-9]*)?/,
-          /su-text-[A-Za-z0-9]*(-)?([A-Za-z0-9]*)?/,
-        ],
-      },
-      // To preserve special characters in Tailwind class names
-      defaultExtractor: (content) => content.match(/[\w-/:]+(?<!:)/g) || [],
-    }),
+    require("tailwindcss"),
+    require("autoprefixer"),
```

Comment:
Noting that I removed the purgecss configuration, as it was causing build issues when used alongside Tailwind. 
My understanding of why this was being included was because we were including the entire compiled stylesheet from Decanter, which was slowing down hot reloading and increasing the size of build files. So this seems like it would now be irrelevant given this change.

But let me know if there's context I'm missing and I can work out how to add it back in.

Reply:
@nplowman In our SAA repo, when we first started implementing the JIT mode, it was on a much earlier version of TW when they first introduced the JIT feature. So we found that we still need those settings otherwise the variant classes (the ones with the :) didn't get generated properly. This might have been fixed in the later version of TW.

Reply:
Looks like we already got rid of this completely in the SAA repo, so definitely safe to remove. Thanks!

Reply:
Yes